config/webpack/webpack.config.test.js

```diff
@@ -28,16 +31,25 @@ module.exports = {
     }
   }),
   // enzyme webpack issue https://github.com/airbnb/enzyme/issues/47
+  // Please note that externals may have to change for versions of React
+  // other than 0.14.x
   externals: {
-    "cheerio": "window",
     "react/lib/ExecutionEnvironment": true,
-    "react/lib/ReactContext": true
+    "react/lib/ReactContext": true,
+    "text-encoding": "window"
   },
   module: _.assign({}, prodCfg.module, {
     // enzyme webpack issue https://github.com/airbnb/enzyme/issues/47
     noParse: [
       /\/sinon\.js/
-    ]
+    ],
+    // enzyme webpack issue https://github.com/airbnb/enzyme/issues/47
+    loaders: (prodCfg.module.loaders || []).concat([
+      {
+        test: /\.json$/,
+        loader: "json"
```

Comment:
And the above comment applies _after_ you've moved `json-loader` to `dev/package.json:dependencies`.